tempesta_fw/classifier.h

```diff
@@ -28,6 +28,11 @@
 #include "tempesta_fw.h"
 #include "connection.h"
 
+/* Size of classifier private cliet accounting data. */
+#define TFW_CLASSIFIER_ACCSZ	256
+
+typedef struct { char _[TFW_CLASSIFIER_ACCSZ]; } TfwClassifierPrvt;
```

Comment:
What guarantees that the actual data will not exceed this value? I think there should be a compile-time check for that somewhere.